gladier_kanzus/tools/prime.py

```diff
@@ -1,25 +1,59 @@
 from gladier import GladierBaseTool, generate_flow_definition
 
-def funcx_prime(**data):
+def dials_prime(**data):
     """Run the PRIME tool on the int-list.
     - Change dir to the <exp>_prime directory
     - Create phil file for prime.run
-    - Call prime.run and pipe the log into a file using the input_range
-    - cp the prime's log into the images dir
-    - zip the prime dir and copy that into the images dir"""
+    - Call prime.run with the current list of ints generated
+    - (not implemented) cp the prime's log into the images dir
+    - (not implemented) zip the prime dir and copy that into the images dir
+    Variables:
+    - data['proc_dir'] is the path to where dials will run and save results
+    - data['prime_dir'] is the path to the folder which prime will run and save results
+    - data['exp'] is the experiment name
+    - data['run_num'] is the beamline run associated with this sample.
+    Optional Variables:
+    - data['unit_cell'] sets a new unit cell into the prime-phil
+    - data['prime_dmin'] sets the dmin value different than 2.1
+    - data['dials_path'] can be set for different installations (the container always have a link at '/')
```

Comment:
This is really good! It helps to know what is being used where. I'd be curious what the implications of changing some of the values are though. For instance, what happens if `prime_dmin` is larger than 2.1?